### PR TITLE
WIP[DEVX-5443] Add forceMuteStream and forceMuteAll

### PR DESCRIPTION
**Pull request by kellyjandrews**
#### What is this PR doing?
This PR serves as the implementation of `forceMuteStream` and `forceMuteAll` methods.

#### What are the relevant tickets?
https://jira.vonage.com/browse/DEVX-5443?src=confmacro

### CODE REVIEW

File: lib/opentok.js [hunk 1]
```diff
@@ -995,6 +995,106 @@ OpenTok = function (apiKey, apiSecret, env) {
...
+  }, function cb(err) {
+    if (err) { return callback(err); }
+    return callback(null);
+  });
+};
+
+/**
+   * Force mute a specific stream in a session with a callback.
+   *
+   * @param sessionId The session ID corresponding to the session where muting a stream.
```
Comment (jeffswartz) [inline lib/opentok.js:1043]:
```suggestion
   * @param sessionId The session ID.
```

File: lib/opentok.js [hunk 2]
```diff
@@ -995,6 +995,106 @@ OpenTok = function (apiKey, apiSecret, env) {
...
+    sessionId: sessionId,
+    streamId: streamId
+  }, function cb(err) {
+    if (err) { return callback(err); }
+    return callback(null);
+  });
+};
+
+/**
+   * Force mute a specific stream in a session with a callback.
```
Comment (jeffswartz) [inline lib/opentok.js:1041]:
```suggestion
   * Force all streams (except for an optional array of streams) in a session to mute published audio.
```

File: lib/opentok.js [hunk 3]
```diff
@@ -995,6 +995,106 @@ OpenTok = function (apiKey, apiSecret, env) {
...
+   * @param sessionId The session ID corresponding to the session where muting a stream.
+   *
+   * @param opts The stream ID corresponding to the stream to be muted.
+   * <ul>
+   *
+   *   <li>
+   *     <code>excudedStreamIds</code> &mdash; An array of Stream IDs to exlude from mute.
+   *   </li>
+   *
+   * </ul>
```
Comment (jeffswartz) [inline lib/opentok.js:1052]: This `options` object should have two properties:

* `excludedStreamIds` (Array) -- An array of stream IDs to be excluded from being muted.
* `active` (Boolean) -- Whether streams published after call to the method will have audio muted (`true`) or not (`false`).

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: lib/opentok.js (hunk 3 above)
Comment (moficodes) [reply lib/opentok.js:1084]: @jeffswartz

**Review by jeffswartz — APPROVED** [review]

**Comment by kellyjandrews** [comment]
@jeffswartz should I merge? Sorry - not sure what the protocol is :)

**Pull request merged**